### PR TITLE
Add Anti-DDoS configs module

### PR DESCRIPTION
**Pull request by irina-pereiaslavskaia**

### CODE REVIEW

**Comment by irina-pereiaslavskaia** [comment]
recheck

File: plugins/modules/anti_ddos_configs_info.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+---
+module: anti_ddos_configs_info
```
Comment (anton-sidelnikov) [inline plugins/modules/anti_ddos_configs_info.py:16]: anti_ddos_optional_policies_info

File: plugins/modules/anti_ddos_configs_info.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+short_description: Get Anti-DDoS defense policies info
```
Comment (anton-sidelnikov) [inline plugins/modules/anti_ddos_configs_info.py:17]: Get Anti-DDoS optional defense policies info

File: plugins/modules/anti_ddos_configs_info.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.2.1"
+author: "Irina Pereiaslavskaia (@irina-pereiaslavskaia)"
+description:
+  - Get optional Anti-DDoS defense policies from the OTC.
+options: {}
```
Comment (anton-sidelnikov) [inline plugins/modules/anti_ddos_configs_info.py:23]: i think you can skip this

File: plugins/modules/anti_ddos_configs_info.py [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+requirements: ["openstacksdk", "otcextensions"]
+'''
+
+RETURN = '''
+anti_ddos_configs_info:
```
Comment (anton-sidelnikov) [inline plugins/modules/anti_ddos_configs_info.py:28]: anti_ddos_optional_policies_info

File: plugins/modules/anti_ddos_configs_info.py [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+          sample: 2000
+        setID:
+          description: Position ID
+          type: int
+          sample: 33
+'''
+
+EXAMPLES = '''
+# Querying Optional Anti-DDoS Defense Policies
+- opentelekomcloud.cloud.anti_ddos_configs_info:
```
Comment (anton-sidelnikov) [inline plugins/modules/anti_ddos_configs_info.py:113]: - opentelekomcloud.cloud.anti_ddos_optional_policies_info

File: plugins/modules/anti_ddos_configs_info.py [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    def run(self):
+
+        data = []
+
+        for raw in self.conn.anti_ddos.configs():
+            dt = raw.to_dict()
+            dt.pop('location')
+            data.append(dt)
+
+        self.exit(changed=False, anti_ddos_configs_info=data)
```
Comment (anton-sidelnikov) [inline plugins/modules/anti_ddos_configs_info.py:132]: anti_ddos_optional_configs_info

File: tests/integration/targets/anti_ddos_configs_info/tasks/main.yaml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+- module_defaults:
+    opentelekomcloud.cloud.anti_ddos_configs_info:
+      cloud: "{{ test_cloud }}"
+  block:
+    - name: Get Anti-DDoS Floating IPs statuses
```
Comment (anton-sidelnikov) [inline tests/integration/targets/anti_ddos_configs_info/tasks/main.yaml:6]: need proper name to step

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: plugins/modules/anti_ddos_configs_info.py (hunk 1 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/anti_ddos_configs_info.py]: Done

File: plugins/modules/anti_ddos_configs_info.py (hunk 4 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/anti_ddos_configs_info.py]: Done

File: plugins/modules/anti_ddos_configs_info.py (hunk 6 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/anti_ddos_configs_info.py]: Done

File: plugins/modules/anti_ddos_configs_info.py (hunk 5 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/anti_ddos_configs_info.py]: Done

File: tests/integration/targets/anti_ddos_configs_info/tasks/main.yaml (hunk 7 above)
Comment (irina-pereiaslavskaia) [reply tests/integration/targets/anti_ddos_configs_info/tasks/main.yaml]: Done

File: plugins/modules/anti_ddos_optional_policies_info.py [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+---
+module: anti_ddos_optional_policies_info
+short_description: Get Anti-DDoS optional defense policies info
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.2.1"
```
Comment (tischrei) [inline plugins/modules/anti_ddos_optional_policies_info.py:19]: check current version

File: plugins/modules/anti_ddos_configs_info.py (hunk 2 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/anti_ddos_configs_info.py]: Done

**Review by tischrei — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

File: plugins/modules/anti_ddos_optional_policies_info.py (hunk 8 above)
Comment (tischrei) [reply plugins/modules/anti_ddos_optional_policies_info.py:19]: it is 4.0

Comment (irina-pereiaslavskaia) [reply plugins/modules/anti_ddos_optional_policies_info.py]: Done

**Review by anton-sidelnikov — APPROVED** [review]

**Review by tischrei — APPROVED** [review]

**Pull request merged**